fic_modules/markdown_modules.py

```diff
@@ -269,7 +275,8 @@ def create_hg_md_table(repository_name):
         md_file.close()
     except FileNotFoundError:
         if LOGGER:
-            print("Json for {} is empty! Skipping!".format(repository_name))
+            LOGGER.error("Json for {} is empty! Skipping!".format(repository_name))
+            # print("Json for {} is empty! Skipping!".format(repository_name))
```

Comment:
If we don't require this anymore, it should be removed.

Reply:
Fixed.